src/platform/packages/private/kbn-language-documentation/src/components/shared/documentation_content.tsx

```diff
@@ -10,16 +10,58 @@ import React from 'react';
 import { css } from '@emotion/react';
 import { i18n } from '@kbn/i18n';
 import { useEuiTheme, euiScrollBarStyles } from '@elastic/eui';
-import { EuiFlexGroup, EuiText, EuiBetaBadge } from '@elastic/eui';
+import { EuiFlexGroup, EuiFlexItem, EuiText, EuiBetaBadge } from '@elastic/eui';
 import type { LanguageDocumentationSections } from '../../types';
 
+interface LicenseInfo {
+  name: string;
+  isSignatureSpecific?: boolean;
+  paramsWithLicense?: string[];
+}
+
+interface MultipleLicenseInfo {
+  licenses: LicenseInfo[];
+  hasMultipleLicenses: boolean;
+}
+
+// Helper function to get licenses array from either format
+function getLicensesArray(license: MultipleLicenseInfo | undefined): LicenseInfo[] {
+  if (license && Array.isArray(license.licenses)) {
+    return license.licenses;
+  }
+  return [];
+}
+
+function createLicenseTooltip(license: LicenseInfo): string {
+  let tooltip = i18n.translate('languageDocumentation.licenseRequiredTooltip', {
+    defaultMessage: 'This feature requires {license} license',
```

Comment:
Super ideally these sentences would say:
"This feature requires a Platinum subscription."
"This feature requires an Enterprise subscription."

- We should use "subscription" instead of "license" (that's the term we use on the website and in the product UI)
- It feels a lot more natural with "a" or "an" before the subscription name. Is this possible to add this logic?